README.md

```diff
@@ -21,4 +21,6 @@ In the root directory of the project, type `docker compose up --build --force-re
 
 The database migrations should be automatically applied by Docker when it spins up the containers. If you are receiving errors during this step, it is best to delete any old containers and volumes you have lying around to ensure that Docker creates the setup form a clean state. 
 
+> We have noticed that the containers do not attach properly if a user is running docker through the `mnt/c/` directory when using WSL. Please have the project placed directly in your host filesystem and spin up the containers directly from your host OS. 
```

Comment:
I recently discovered GitHub Flavored Markdown supports special highlighting for alerts, so we can do something like this:

```md
> [!IMPORTANT]
> We have noticed yada yada yada...
```
->
> [!IMPORTANT]
> We have noticed yada yada yada...

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts